json_yaml_files/prepare_files.py

```diff
@@ -0,0 +1,178 @@
+from bids import BIDSLayout
+from collections import defaultdict
+import re
+import json
+import sys
+import yaml
+from pathlib import Path
+
+
+class BIDStoNDAConfigGenerator:
+    def __init__(self):
+        self.path = sys.argv[1]
+        self.layout = BIDSLayout(self.path, derivatives=True)
```

Comment:
Since we are not doing derivatives at first, should this just be `False` for now?

```suggestion
        self.layout = BIDSLayout(self.path, derivatives=False)
```

Reply:
sure. Since we are not handling derivatives, we can set it to false.